Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM python:2.7-slim
+
+MAINTAINER Azavea
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    g++ \
+    git \
+    gfortran \
+    libatlas-base-dev \
+    libblas-dev \
+    libgeos-dev \
+    liblapack-dev \
+    libopenmpi-dev \
+    libtool \
+    openmpi-bin \
+    python-all-dev \
+    python-pip \
+    wget
+
+# Download, build, and install GDAL
+RUN cd tmp && \
```

Comment:
Usually the convention is to place source code like this that is planned to be compiled into `/usr/local/src`.